api/v1/serializers/pets-serializer.js

```diff
@@ -14,54 +15,68 @@ const petResourcePath = 'pets';
 
 /**
  * The column name getting from database is usually UPPER_CASE.
- * This block of code is to make the camelCase keys defined in swagger.yaml be
+ * This block of code is to make the camelCase keys defined in openapi.yaml be
  * UPPER_CASE so that the serializer can correctly match the corresponding columns
  * from the raw data rows.
  */
 _.forEach(petResourceKeys, (key, index) => {
   petResourceKeys[index] = decamelize(key).toUpperCase();
 });
 
-const serializerArgs = {
-  identifierField: 'ID',
-  resourceKeys: petResourceKeys,
-};
-
 /**
  * @summary Serializer petResources to JSON API
  * @function
- * @param {[Object]} rawPets Raw data rows from datasource
+ * @param {[Object]} rawPets Raw data rows from data source
  * @param {Object} query Query parameters
  * @returns {Object} Serialized petResources object
  */
 const SerializedPets = (rawPets, query) => {
+  const serializerArgs = {
+    identifierField: 'ID',
+    resourceKeys: petResourceKeys,
+  };
+
   /**
    * Add pagination links and meta information to options if pagination is enabled
    */
-  const { page } = query;
-  if (page) {
-    const pagination = paginate(rawPets, page);
+  const pageQuery = {
+    size: query['page[size]'],
+    number: query['page[number]'],
+  };
+
+  if (pageQuery.size || pageQuery.number) {
```

Comment:
Won't this condition always be true since the default values are handled by a package now?

Reply:
Right, good catch.